recipes/autodiff/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
You can delete this file as it's already included in the tarball.

Reply:
@chrisburr Thanks for reviewing this recipe! I'll delete this file.